### ISSUE TITLE
Draft 2020-12 

### ISSUE THREAD

**Issue by kilianmh** [fisxoj/json-schema#24]
Json Schema draft 2020-12 is not mentioned in the docs. Is this version supported, or not yet?

**Comment by fisxoj** [comment]
Yes, it's not currently supported. I'm not familiar with the changes in the new draft to know how much work adding it would take.

At this point, there's a strong enough pattern to follow for new versions unless it introduces functionality that would be hard with the current setup.

I can't promise much, these days, but prs are always welcome.
